libs/postgres.js

```diff
@@ -0,0 +1,14 @@
+const { Client } = require('pg');
+
+
+const USER = encodeURIComponent(config.dbUser);
+const PWD = encodeURIComponent(config.dbPassword);
+const uri = `postgres://${USER}:${PWD}@${config.dbHost}:${config.dbPort}/${config.dbName}`;
+
+async function getConnection() {
```

Comment:
The function will create a new connection when you call it each time. Maybe you could use a variable to store the last client and use it after the first connection.